numpyro/infer/mcmc.py

```diff
@@ -593,6 +593,14 @@ def get_samples(self, group_by_chain=False):
             `dict` keyed on site names if a model containing Pyro primitives is used,
             but can be any :func:`jaxlib.pytree`, more generally (e.g. when defining a
             `potential_fn` for HMC that takes `list` args).
+
+        **Example:**
+
+        You can then pass those samples to :class:`~numpyro.infer.util.Predictive`:
+
+            posterior_samples = mcmc.get_samples()
+            predictive = Predictive(model, posterior_samples=posterior_samples)
+            samples = predictive(rng_key1, *model_args, **model_kwargs)
```

Comment:
I think you need an empty string after this to make lint pass.

Reply:
I think it might be the `*` - will try making it a `r` string

Reply:
OK, fixed, it just needed to be put into a code block, so with the previous line ending with `::` rather than `:`

Reply:
now it looks like this:

![Screenshot 2021-07-11 at 21 17 21](https://user-images.githubusercontent.com/33491632/125208931-6e13f580-e28d-11eb-9202-ebb038f329e2.png)

Reply:
yeah, good catch!